docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
   app:
     build:
       context: .
-      dockerfile: ./docker/app/Dockerfile
+      dockerfile: ./docker/base/Dockerfile
```

Comment:
I'm a bit confused by this, the `app` directory mentioned here doesn't exist.
Please let me know if I have missed something.